### DISCUSSION TITLE
Running Pipeline Favour

### DISCUSSION THREAD

**Discussion by helena-shapton** [blex-max/ecto_pipe#18] [General]
Hi Alex,

I have the remaining plate sequences now and I'm really intrigued to see the results Re: your meeting with Jim yesterday. I have tried to contact Jim and Laura all day to assist but no luck so could you possibly run the following and send me all the outputs (inc. raw OTU cluster fasta for all sequences). Zip file is being a pain and not uploading so I have sent to your gmail. Also I a different file tag and output folder but I still couldn't get the sequences to assemble.

Helena

**Comment by blex-max** [comment]
Sure - can you send me a zip with all the traces you want done, including
ones I've done for you before?

**Reply by helena-shapton** [reply]
Sent to both of your emails it won't attach on git for some reason

**Comment by helena-shapton** [comment]
Shapton, Helena has shared a OneDrive for Business file with you. To view it, click the link below.
<https://imperiallondon-my.sharepoint.com/personal/hs1019_ic_ac_uk/Documents/Attachments/ITS_sequences%205.zip>
[https://r1.res.office365.com/owa/prem/images/dc-zip_20.png]<https://imperiallondon-my.sharepoint.com/personal/hs1019_ic_ac_uk/Documents/Attachments/ITS_sequences%205.zip>
ITS_sequences 5.zip<https://imperiallondon-my.sharepoint.com/personal/hs1019_ic_ac_uk/Documents/Attachments/ITS_sequences%205.zip>



Helena Shapton
Final Year
Biological Sciences

Imperial College London
South Kensington Campus
London SW7 2AZ, UK

Tel: (+44) 07891 668811

________________________________

**Comment by helena-shapton** [comment]
To view ITS_sequences 5.zip, sign in<https://imperiallondon-my.sharepoint.com/personal/hs1019_ic_ac_uk/_layouts/15/acceptinvite.aspx?invitation=%7B39658DD6%2D45A1%2D4609%2D9CA3%2DE5C13B8FE736%7D&listId=245f8dc6%2D8cff%2D42f0%2Daf1b%2D8c7486e6da82&itemId=b431377f%2Df81a%2D473c%2Da248%2D671d9f6a22f5> or create an account.

**Comment by blex-max** [comment]
I can't access any of those - can you send it as a google drive link to my gmail perhaps?

**Reply by helena-shapton** [reply]
Sent

**Reply by helena-shapton** [reply]
Any updates?

**Comment by blex-max** [comment]
Done

[helena.15_05_22.zip](https://github.com/kew-myco/its_glue/files/8695057/helena.15_05_22.zip)

**Comment by helena-shapton** [comment]
Thank you

If possible, Laura also believes there to be a file available that assigns an SH number to all OTUs which could be usful to use?

Helena Shapton
Final Year
Biological Sciences

Imperial College London
South Kensington Campus
London SW7 2AZ, UK

Tel: (+44) 07891 668811

________________________________

**Reply by blex-max** [reply]
That file doesn't strictly exist the way you've put it, but there is a file of attempted sintax classifications for every sequence, including best matches below the sintax cutoff to accept as 'true'. I think this is probably what you mean, so attached. Tab-separated values. If you needed to match these to clusters, you can use this in combo with the cluster membership table. But a quick warning: Using below-cutoff matches entirely defeats the point of taxonomy assignment algorithm with taxonomic-rank confidence scores. You will introduce a lot of over-classification errors.
[sintax_classifications.tsv](https://github.com/kew-myco/its_glue/files/8698732/sintax_classifications.csv)
